core/drivers/crypto/se050/adaptors/sub.mk

```diff
@@ -1,5 +1,6 @@
 cflags-y += -Wno-strict-aliasing
 cflags-y += -DAX_EMBEDDED=1
+cflags-y += -DUSE_RTOS=0
```

Comment:
Would it be worth moving the common flags to a common file? For example create `core/drivers/crypto/se050/cflags.mk` and here or there `include cflags.mk` or `include ../cflags.mk`. Then you only need to update `cflags.mk`.

Reply:
sounds good, will do that.
will also do the release.tarball, modify shippable and update the PR accordingly, better that way